### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Prashantrajput11**
Added homepage for ecom

Hero section preview :

image
Faetured Products preview : 

image

image

### CODE REVIEW

File: css/homepage.css [hunk 1]
```diff
@@ -0,0 +1,118 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600&family=Poppins:wght@300;400;500&display=swap');
+
+html {
+  font-size: 62.5%;
+  font-family: 'Poppins', sans-serif;
+}
+
+/*********************/
+/*****Utilties*******/
+/********************/
```
Comment (manojasarsa) [inline css/homepage.css:10]: You can use a single line for comments.

File: css/homepage.css [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+.flex-vt-sp {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+
+  /* border: 2px solid red; */
+  /* margin: 2rem 0; */
```
Comment (manojasarsa) [inline css/homepage.css]: You can remove these comments if no longer needed.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+						<a href="" class="btn  secondary-cta--card card-cta-btn">Save to wishlist</a>
+					</div>
+				</div>
+			</div>
+			<!-- card3 -->
+
+		</div>
+
+
+	</section>
```
Comment (manojasarsa) [inline index.html:126]: Avoid too much spacing

**Review by manojasarsa — APPROVED** [review]
Rest looks good!

File: css/homepage.css [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+}
+
+.btn--login {
+  background: var(--secondary-color);
+  border-radius: 0;
+  color: #4d4d4d;
+}
+
+.btn--login:hover {
+	background: #b8b8b8;
```
Comment (khushijohri2001) [inline css/homepage.css]: You can remove the extra spaces and unwanted comments. Otherwise code looks good to me

**Review by khushijohri2001 — CHANGES_REQUESTED** [review]

**Pull request merged**